template_content_swapper/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module provides a generic functionality to replace QWeb view elements.
```

Comment:
It would also be nice to add a `USAGE.rst` section where the screenshots / example from @yostashiro is added. This will then be added to the `index.html` of the module and it will be instantly clear to see how the module could be used.